nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_with_audio.py

```diff
@@ -169,7 +181,7 @@ def __init__(
                 abbreviation_graph = AbbreviationFst(whitelist=whitelist, deterministic=deterministic).fst
                 classify_and_verbalize |= pynutil.add_weight(pynini.compose(abbreviation_graph, v_abbreviation), 100)
 
-            punct_only = pynutil.add_weight(punct_graph, weight=2.1)
+            punct_only = pynutil.add_weight(punct_graph, weight=20.1)
```

Comment:
can the weights in the file be the same as audio_with_lm.py? i.e. [1,2,100]? this would make it cleaner

Reply:
yes: TBD